gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 # Required to keep catalog compatibility with 3.4.x.  Can be removed for 4.0.0
-managed-neo4j-compat = "3.5.34"
+managed-neo4j-compat = "4.4.10"
 managed-neo4j-driver-compat = "4.4.9"
 
 managed-neo4j = "3.5.34"
```

Comment:
This needs to be updated as well, to keep it the same as the `compat` one

As this is a major update, it probably has to be a major release of this module, and wait for Micronaut 4.0.0?